### PR TITLE
[Model Runner V2] Migration to support quantized model by default [5/N]

### PR DESCRIPTION
**Pull request by yewentao256**
## Purpose

Part of https://github.com/vllm-project/vllm/issues/41286

Tests coverd by CI

### CODE REVIEW

**Review by yewentao256 — COMMENTED** [review]
[buildkite/ci/pr/attention-benchmarks-smoke-test-b200](https://buildkite.com/vllm/ci/builds/72155#019ecdda-20b1-49fd-8fc7-f2aa195895f1) has been fixed in main

[buildkite/ci/pr/amd-kernels-quantization-test-2-mi325-1](https://buildkite.com/vllm/ci/builds/72155#019ecdda-21ff-4913-91d2-d00b2edec666)
[buildkite/ci/pr/amd-v1-sample-plus-logits-mi325-1](https://buildkite.com/vllm/ci/builds/72155#019ecdda-2275-4284-89b2-857dd11562d7)

are unrelated, we can remove the run all tests label now.

**Review by njhill — APPROVED** [review]
